scala-thoughts.md

```diff
@@ -0,0 +1,29 @@
+###### What is easy to do in Scala? What's not?
+Easy:
+- String concatenation
+    - Strings can be added to other strings, and strings can be concatenated from a List form.
+- Type checking/aliasing
+    - I love static typing - especially when running Scala in codespaces can cause issues. I also find it cool that you can name a custom type (ex. Color, Board in Mastermind) and I can see how this would help with  readability of objects in domain-specific programs.
+
```

Comment:
I liked the types too! It's nice when you can make types and type alias make sense like it did in Mastermind.